registry/storage/driver/s3/s3_test.go

```diff
@@ -69,6 +69,7 @@ func init() {
 			v4AuthBool,
 			minChunkSize,
 			rootDirectory,
+			"",
```

Comment:
Maybe something for a separate PR, but I would really like to see this statement use named initializers instead of relying on the order of the fields.